Jenkinsfile

```diff
@@ -235,16 +246,37 @@ pipeline {
                         withCredentials([usernamePassword(credentialsId: 'openshift-nexus',
                             passwordVariable: 'NXPASS', usernameVariable: 'NXUSER')]) {
                             sh """
-                            echo $NXPASS | podman login ${Globals.IMAGE_REPO_INTERN} -u $NXUSER --password-stdin
+                            echo $NXPASS | podman login ${Globals.NEXUS_IMAGE_REPO_INTERN} -u $NXUSER --password-stdin
                             podman push ${Globals.imageTagIntern}
                             """
                             if (env.BRANCH_NAME == 'main'){
                                 sh """
-                                echo $NXPASS | podman login ${Globals.IMAGE_REPO_PUBLIC} -u $NXUSER --password-stdin
+                                echo $NXPASS | podman login ${Globals.NEXUS_IMAGE_REPO_PUBLIC} -u $NXUSER --password-stdin
                                 podman push ${Globals.imageTagPublic}
                                 """
                             }
-                        }  
+                        }
+                        if (params.PUSH_IMAGES_TO_ECR) {
+                            withCredentials([usernamePassword(credentialsId: 'aws-icon-sandbox',
+                                                        passwordVariable: 'AWS_SECRET_ACCESS_KEY',
+                                                        usernameVariable: 'AWS_ACCESS_KEY_ID')]) {
+                            echo "---- PUBLISH IMAGE TO ECR ----"
+                            sh """
+                            #!/bin/bash
+
+                            if test -f /etc/ssl/certs/ca-certificates.crt; then
+                                export AWS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
+                            else
+                                export AWS_CA_BUNDLE=/etc/ssl/certs/ca-bundle.crt
+                            fi
+                            aws ecr get-login-password --region eu-central-2 | podman login --username AWS --password-stdin --cert-dir /etc/ssl/certs ${ECR_REPO}
+
+                            podman push --cert-dir /etc/ssl/certs ${Globals.imageTagECR}
+                            aws ssm put-parameter --name "/flexpart_ifs/flexprep/containertag" --type "String" --value "${TAG}" --overwrite --region eu-central-2
```

Comment:
Storing this parameter is of little value if it does not correspond to the tag of the image that is pushed

Reply:
I updated the tag pushed to correspond to that one TAG.